components/cam/src/dynamics/se/dyn_comp.F90

```diff
@@ -417,40 +417,40 @@ subroutine write_grid_mapping(par, elem)
 
     ! Create a CS grid mapping file for postprocessing tools
 
-       ! write meta data for physics on GLL nodes
-       call cam_pio_createfile(nc, 'SEMapping.nc', 0)
-   
-       ierr = pio_def_dim(nc, 'ncenters', npm12*nelem, dim1)
-       ierr = pio_def_dim(nc, 'ncorners', 4, dim2)
-       ierr = pio_def_var(nc, 'element_corners', PIO_INT, (/dim1,dim2/),vid)
-    
-       ierr = pio_enddef(nc)
-       if (par%dynproc) then
-          call createmetadata(par, elem, subelement_corners)
-       end if
+    ! write meta data for physics on GLL nodes
+    call cam_pio_createfile(nc, 'SEMapping.nc')
 
-       jj=0
```

Comment:
It would be better to move the whitespace changes to a separate commit (easier to view changes on github without "-w")

Reply:
Do the last two commits I just pushed fix this? Or do I need to alter the git history somehow to make this play nice?

Reply:
Yeah rebasing the branch (two commits, the second commit does not include any white space changes. If you want to include the white space changes add it as a third commit with just the white space changes) would be nicer. Let me know if you need help with it.

Reply:
Thanks the rebased branch looks good

Reply:
No problem!